src/api/routes/interviews.ts

```diff
@@ -208,13 +209,29 @@ const getInterviewerStats = procedure
       return acc;
```

Comment:
At line 206, comment on the structure of `interviewCountMap` (e.g. "a map of ....")